pkg/truncate/truncate.go

```diff
@@ -3,30 +3,40 @@ package truncate
 
 import "fmt"
 
+const truncateFormat = "XXXtruncated%dbytesXXX"
```

Comment:
Did we ever understand why this was required?

Reply:
Unfortunately not. I had a look on the surrounding code without understanding it, and decided not to spend more time on it.